_posts/2023-02-20-intelligent-automation.md

```diff
@@ -0,0 +1,223 @@
+These business experts have accumulated an immeasurable wealth of knowledge and expertise that constantly faces an inevitable expiration date (be it pensions or even turnover). 
+Safeguarding this knowledge through proper knowledge management often is a set of processes on its own to which intelligent automation can once again be applied.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2023-02-20-intelligent-automation/rpa-ai-balance.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+For RPA, the biggest gains will be attained by using the robots as a stepping stone to digital maturity in your process automation. 
+They are tools to be used to quickly replace error-prone human tasks and capitalize on the quick gains this provides. 
+They can also be utilized to quickly set up proof-of-concept endeavors to prove added value derived from ideation. 
+Once these steps in the automated process reach the proper level of maturity, they can then more easily be replaced by technologies that trade in the hyper agility RPA provides for more grip on the governance and maintainability that API solutions offer. 
+Camunda specified a three-step approach along those lines in their 2020 White Paper titled “Beyond RPA: How to Build Toward End-to-End Process Automation”. 
+These steps are the following:
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2023-02-20-intelligent-automation/rpause.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+A big part of the added value realized by introducing AI robots into the process automation toolbox is knowing where and when to use them. 
+Predictive models can help decisioning in organizations, but there are some points to keep in mind:
+- At its core effective decisioning should answer not only answer the “What is Happening?” question, but also the “Why is it happening?”-question. This understanding of the reasons behind information leads to better opportunity detection and initiatives.
+- Risk/Reward considerations should be available with each option of the decisions.
+- Even with decisions where there are ambiguous or even unknown factors, or where the risk/reward cannot be determined, there is a need to automate, and this is where deep learning/machine learning models can make the difference.
+
+The types of tasks that RPA and AI can alleviate are then:
+- Tasks that are easy to do when there are few, but become cumbersome in large amounts. 
+- Tasks that are not easy for humans to perform, such as high-speed or high-complexity algorithms.
+- Tasks that require the interpretation and/or parsing of large amounts of knowledge (such as a volume of laws or a medical database).
+
+Although the degree of independence with which robots can execute these tasks varies widely from case to case. 
```

Comment:
Shouldn't this and the next line be 1 sentence?

Reply:
no, this is correct